src/tooltip.tsx

```diff
@@ -116,6 +117,8 @@ const Tooltip: React.FC<Props> = ({
     targetLabel,
     delay = true,
     dataAttributes,
+    fullWidth,
+    changedPosition,
```

Comment:
This property is a string because it can receive a dynamic position or another position that you want to set, like 20 px, where you can change the position of the Tooltip.

We created this prop for the use case like the PR of the Slider, we would need to be able to dynamically change the position of the Tooltip

Reply:
I understand the motivation behind this prop but this this shouldn't be needed with a correctly implemented tooltip (although refactoring it is out of scope of this task). This prop is not the best solution but until we rewrite the component, this will help us moving forward workarounding tooltip issues.

Please, rename this prop to something like `unstable_offsetX` or `unstable_leftPosition`. The type of this prop should be `number` (not string) and that number will be added to the initial X position to be able to move it (this `unstable_` prefix is a react convention to name things that will change or be removed in the future. Add the eslint ignore comment if needed).

This component currently has many issues that should be addressed in a different task:
* bad positioning:
![image](https://github.com/Telefonica/mistica-web/assets/579417/cdb77024-de51-493b-9a84-3946b5e4c1e7)
* strange interactions (works with hover/click and both implementations collide)
* there should be a mechanism to prevent showing two tooltips at the same time
* perhaps we should implement new interactions, like showing on tap and hide on tap release
* Position the tooltip absolutely to a relative target, so if the target moves (by scrolling the page or sliding it), the tooltip will automatically move. [Example here](https://codepen.io/pladaria/pen/BavBWaj)

Reply:
Agree with this approach to move forward with this RP

Reply:
@BrunoGuimaraes103 please, read my previous comment